desktop/cmp/dash/DashViewSpec.ts

```diff
@@ -53,6 +53,15 @@ export interface DashViewSpec {
     /** True (default) to allow renaming the view. */
     allowRename?: boolean;
 
-    /** Additional properties to store on the DashView */
-    [x: string]: any;
+    /**
```

Comment:
I’m pretty sure `renderMode` and `refreshMode` are specific to `DashContainerViews` (and already exist in the `DashContainerViewSpec` interface which extends this one):

https://github.com/xh/hoist-react/blob/4f33c374c184d4d28acfb0b6226dc377eab12cd7/desktop/cmp/dash/container/DashContainerViewSpec.ts#L13